<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="leftColumn">
        <item>
         <widget class="QLabel" name="srcCode_lbl">
          <property name="text">
           <string>Исходный текст</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="srcCode">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="codeTable_lbl">
          <property name="text">
           <string>Таблица кодов операций</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="codeTable">
          <property name="cursor" stdset="0">
           <cursorShape>CrossCursor</cursorShape>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>9</number>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
          <column/>
          <item row="0" column="0">
           <property name="text">
            <string>LOAD</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>LOADI</string>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item row="1" column="2">
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>STORE</string>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item row="2" column="2">
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>ADD</string>
           </property>
          </item>
          <item row="3" column="1">
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item row="3" column="2">
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>SUB</string>
           </property>
          </item>
          <item row="4" column="1">
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item row="4" column="2">
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item row="5" column="0">
           <property name="text">
            <string>JMP</string>
           </property>
          </item>
          <item row="5" column="1">
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item row="5" column="2">
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item row="6" column="0">
           <property name="text">
            <string>JZ</string>
           </property>
          </item>
          <item row="6" column="1">
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item row="6" column="2">
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item row="7" column="0">
           <property name="text">
            <string>HALT</string>
           </property>
          </item>
          <item row="7" column="1">
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item row="7" column="2">
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item row="8" column="0">
           <property name="text">
            <string>NOP</string>
           </property>
          </item>
          <item row="8" column="1">
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item row="8" column="2">
           <property name="text">
            <string>1</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="middleColumn">
        <item>
         <widget class="QLabel" name="helpTable_lbl">
          <property name="text">
           <string>Вспомогательная таблица</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="helpTable">
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="symbolicNameTable_lbl">
          <property name="text">
           <string>Таблица символических имен</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="symbolicNameTable">
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="firstPassErr_lbl">
          <property name="text">
           <string>Ошибки первого прохода</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="firstPassErr"/>
        </item>
        <item>
         <widget class="QPushButton" name="clearFirst">
          <property name="text">
           <string>Очистка ошибок первого прохода</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightColumn">
        <item>
         <widget class="QLabel" name="binaryCode_lbl">
          <property name="text">
           <string>Двоичный код</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="binaryCode"/>
        </item>
        <item>
         <widget class="QLabel" name="secondPassErr_lbl">
          <property name="text">
           <string>Ошибки второго прохода</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="secondPassErr"/>
        </item>
        <item>
         <widget class="QPushButton" name="clearSecond">
          <property name="text">
           <string>Очистка ошибок второго прохода</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="firstPass">
        <property name="text">
         <string>Первый проход</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="secondPass">
        <property name="text">
         <string>Второй проход</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="chooseAddrMethod">
      <item>
       <property name="text">
        <string>Прямая адресация</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Относительная адресация</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Смешанная адресация</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
